test/rx_ruby/core/test_observable_creation.rb

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
 
-require 'test_helper'
+require "#{File.dirname(__FILE__)}/../../test_helper"
```

Comment:
This is not necessary, but it allows for debugging in RubyMine and running the tests with gem called `m`.